lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -228,18 +241,28 @@ def __init__(
                 else self.distance_to_surface * 1e-6,  # um => m
             )
         else:
-            if self.distance_to_surface:
-                raise NotImplementedError(
-                    "Using a distance to the surface is currently not supported for models without "
-                    "hydrodynamic effects."
+            if distance_to_surface:
+                args = [distance_to_surface * 1e-6, bead_diameter * 1e-6 / 2.0]
```

Comment:
Tuples are a better default for immutable data. They better convey the intent and are also faster (not that performance matters here, but it's a better default).
```suggestion
                args = (distance_to_surface * 1e-6, bead_diameter * 1e-6 / 2.0)
```

Reply:
Good point!